google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -41,16 +41,17 @@ class Pipeline(object):
 from airflow.contrib.operators.bigquery_table_delete_operator import BigQueryTableDeleteOperator
 from airflow.contrib.operators.bigquery_to_bigquery import BigQueryToBigQueryOperator
 from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
-from datetime import datetime, timedelta
+from datetime import timedelta
+from pytz import timezone
 
 """
 
   _default_args_format = """
     'owner': 'Datalab',
     'depends_on_past': False,
     'email': ['{0}'],
-    'start_date': datetime.strptime('{1}', '{3}'),
-    'end_date': datetime.strptime('{2}', '{3}'),
+    'start_date': {1},
```

Comment:
If this is the case, I'd suggest we change this name to `start_date_utc` and drop the `Z` at the end if it's specified.